packages/ember-application/tests/system/bootstrap-test.js

```diff
@@ -25,6 +23,6 @@ moduleFor('Ember.Application with default resolver and autoboot', class extends
 
   ['@test templates in script tags are extracted at application creation'](assert) {
     this.runTask(() => this.createApplication());
-    assert.equal(this.$('#app').text(), 'Hello World!');
+    assert.equal(document.querySelector('#app').textContent, 'Hello World!');
```

Comment:
This can be `getElementById`, but we can/should probably just make `#app` `this.element` so you can just use `this.assertText`